actions/trigger-argo-workflow/cmd/trigger-argo-workflow/argo.go

```diff
@@ -136,6 +153,10 @@ func (a *App) Run(md GitHubActionsMetadata) error {
 		var err error
 		uri, out, err = a.runCmd(md)
 
+		if isFatalError(err) {
```

Comment:
LGTM 🙂 Is there perhaps a way to cover this behaviour with a test?

Reply:
thanks @zerok . added unit test for isFatalError.